app/controllers/ems_common.rb

```diff
@@ -464,6 +464,10 @@ def button
       javascript_redirect :controller         => "cloud_volume",
                           :action             => "new",
                           :storage_manager_id => params[:id]
+    elsif params[:pressed] == "cloud_volume_snapshot_create"
+      javascript_redirect :controller => "cloud_volume",
+                          :action     => "snapshot_new",
+                          :id         => find_checked_ids_with_rbac(CloudVolume).first
```

Comment:
@sasoc Please rename all `find_checked_ids_with_rbac` with the one proposed by @himdel and make a separate PR for this.